packages/webhook-service/src/main.js

```diff
@@ -3,7 +3,8 @@
 const crypto = require('crypto')
 const knex = require('./knex')
 const fs = require('fs')
-const metrics = require('./prometheusMetrics')
+const metrics = require('./observability/prometheusMetrics')
+const { webhookServiceLogger } = require('./observability/logging')
```

Comment:
same thing here - the `webhookService` prefix is redundant considering all of this happens inside the `webhookService`